kafka/src/main/java/org/axonframework/kafka/eventhandling/producer/DefaultProducerFactory.java

```diff
@@ -113,6 +113,25 @@ public ConfirmationMode confirmationMode() {
         return confirmationMode;
     }
 
+    /**
+     * Return an unmodifiable reference to the configuration map for this factory.
+     * Useful for cloning to make a similar factory.
+     *
+     * @return the configs.
+     */
+    public Map<String, Object> configurationProperties() {
+        return Collections.unmodifiableMap(configs);
+    }
+
+    /**
+     * TransactionalId prefix for all producer instances.
```

Comment:
Nit: The function states it's a `transactionId`, whilst the javadoc describes a `transactionalId`. I'd prefer them to be identical, if that at all makes sense.